test/scenarios-bootc/periodics/cos9-src@flannel.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+scenario_create_vms() {
+    prepare_kickstart host1 kickstart-centos.ks.template ""
```

Comment:
We need to fix this line for the test to use the right kickstart. See [this example](https://github.com/openshift/microshift/blob/main/test/scenarios-bootc/periodics/cos9-src%40greenboot.sh#L6C1-L7C1) as a reference.
```suggestion
    prepare_kickstart host1 kickstart-bootc.ks.template cos9-bootc-source
```

Reply:
Done 👍🏼